### PR TITLE
fixes #9 Code structure for pip package

### LINKED ISSUE #9: Code structure for Pip package
Make required code changes as required for a pip library

- Folder with package name: `language-detect`
- `__init__.py` , `main.py` etc as needed
- `pyproject.toml`, `setup.py` etc as needed

### PR DESCRIPTION
**Pull request by glad-e**
fixes #9 
This pr contains the code structure to publish a PyPI package.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    "fuzzywuzzy==0.18.0", 
+    "Levenshtein==0.26.1"
+]
+requires-python = ">=3.8"
+authors = [
+    {name = "Kavitha Raju", email = "Kavitha.Raju@bridgeconn.com"},
+]
+maintainers = [ 
+    {name = "BCS Team"}
+]
```
Comment (kavitharaju) [inline pyproject.toml]: Is this how we used to do? 
Its better to put author as "BCS Team" and in maintainers list add both of our names and emails. For the author, field also give a valid email id, either mine or joel@bridgeconn.com.

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+description = "This is a language detection library using ML models"
+readme = "README.md"
+license = {file = "LICENSE"}
```
Comment (kavitharaju) [inline pyproject.toml]: Can we specify the license as MIT here?

**Review by kavitharaju — COMMENTED** [review]
Have you tried publishing to test.pypi? Are you able to install and test ?

**Comment by glad-e** [comment]
> Have you tried publishing to test.pypi? Are you able to install and test ?

No I didn't publish it to test PyPI yet, I only tested it locally

**Comment by kavitharaju** [comment]
> > Have you tried publishing to test.pypi? Are you able to install and test ?
> 
> No I didn't publish it to test PyPI yet, I only tested it locally

Then please try publishing and testing.

**Pull request merged**